srcpkgs/ProTracker/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'ProTracker'
+pkgname=ProTracker
+version=0.20b
+revision=1
+hostmakedepends="subversion"
+makedepends="SDL2-devel"
+depends="SDL2"
+short_desc="Modern clone of the amiga ProTracker music tracker"
+maintainer="rc-05 <rc23@email.it>"
+license="WTFPL"
+homepage="https://16-bits.org/pt.php"
+
+do_fetch() {
```

Comment:
please use a versioned tarball for the package, it won't be accepted otherwise.

Reply:
The problem is that the readme specifies to do a subversion checkout to get the source code... there isn't a tarball with the source code...

Reply:
We don't accept -git or -svn packages, they must be tarballs.

Reply:
You can get a ziped archive from sourceforge e.g. `https://sourceforge.net/code-snapshots/svn/p/pr/protracker/code/protracker-code-r177-trunk.zip` but there are no tagged versions or so only the commit number. (177 is beta 20)